src/GameGrid.tsx

```diff
@@ -86,16 +89,19 @@ interface LetterProps {
   owner?: PlayerDetails;
   isSelected?: boolean;
   isBombed?: boolean;
+  selectPlayer: (player: number | null) => void;
 }
 export function LetterCell({
   letter,
   owner,
   isSelected = false,
   isBombed = false,
+  selectPlayer,
 }: LetterProps): JSX.Element {
   const colour = owner ? ownerColours[owner.index] : "";
   return (
     <div
+      onClick={() => selectPlayer(isSelected ? null : owner ? owner.index : null)}
```

Comment:
i don't think this needs the `isSelected` guard, since selecting the currently selected player is a no-op anyway, so it reduces down to `onClick={owner ? () => selectPlayer(owner.index) : null}`

Reply:
Maybe I'm misunderstanding but we'd want it to deselect the player when clicking on the selected player's letters (like how when you click on the selected player in the players list) with selectPlayer(null)
Also we'd want to deselect the player when clicking on a tile or letter without an owner, or at least I thought that was a good idea.
no-op will do nothing, not deselect.

Reply:
Oh! I see - yeah, I think that makes sense